packages/core/src/components/ListContainer/ListItem/ListItem.tsx

```diff
@@ -0,0 +1,224 @@
+  tabIndex,
+  ...others
+}: HvListItemProps) => {
+  const {
+    topContainerRef,
+    condensed: condensedContext,
+    disableGutters: disableGuttersContext,
+    interactive: interactiveContext,
+  } = useContext<any>(HvListContext);
+
+  const condensed = condensedProp != null ? condensedProp : condensedContext;
+  const disableGutters =
+    disableGuttersProp != null ? disableGuttersProp : disableGuttersContext;
+  const interactive =
+    interactiveProp != null ? interactiveProp : interactiveContext;
+
+  const handleOnClick = useCallback(
+    (evt) => {
+      if (!disabled) {
+        onClick?.(evt);
+      }
+    },
+    [disabled, onClick]
+  );
+
+  const clonedStartAdornment = useMemo(
+    () =>
+      applyClassNameAndStateToElement(
+        startAdornment,
+        selected,
+        disabled,
+        handleOnClick,
+        clsx("startAdornment", disabled && "disabled"),
+        classes?.startAdornment
+      ),
+    [classes?.startAdornment, disabled, handleOnClick, selected, startAdornment]
+  );
+  const clonedEndAdornment = useMemo(
+    () =>
+      applyClassNameToElement(
+        endAdornment,
+        "endAdornment",
+        classes?.endAdornment
+      ),
+    [classes?.endAdornment, endAdornment]
+  );
+
+  const roleOptionAriaProps =
+    role === "option" || role === "menuitem"
+      ? {
+          "aria-disabled": disabled || undefined,
+          "aria-selected": selected,
+        }
+      : {};
+
+  const listItem = (
+    <StyledListItem
+      id={id}
+      role={role}
+      onClick={handleOnClick}
+      onKeyDown={() => {}}
+      className={clsx(
+        className,
+        classes?.root,
+        !disableGutters && classes?.gutters,
+        !disableGutters && "gutters",
+        condensed && classes?.condensed,
+        condensed && "condensed",
+        interactive && classes?.interactive,
+        interactive && "interactive",
+        selected && classes?.selected,
+        selected && "selected",
+        disabled && classes?.disabled,
+        disabled && "disabled",
+        startAdornment != null && classes?.withStartAdornment,
+        startAdornment != null && "withStartAdornment",
+        endAdornment != null && classes?.withEndAdornment,
+        endAdornment != null && "withEndAdornment"
+      )}
+      $gutters={!disableGutters}
+      $interactive={interactive}
+      $disabled={disabled || false}
+      $selected={selected || false}
+      $startAdornment={startAdornment != null}
+      $endAdornment={endAdornment != null}
+      {...roleOptionAriaProps}
+      {...others}
+    >
+      {clonedStartAdornment}
+      {children}
+      {clonedEndAdornment}
+    </StyledListItem>
+  );
+
+  return interactiveProp ? (
+    <HvFocus
+      rootRef={topContainerRef}
+      selected={selected}
+      disabledClass={disabled || undefined}
+      strategy={role === "option" ? "listbox" : "menu"}
+      classes={{ focus: `background-color: ${theme.colors.atmo3}; z-index: 2` }}
```

Comment:
I think we also need to pass `classes.focus`. Otherwise, the users won't be able to override or extend the focus style.

Reply:
ah, you're right.